move_period_from_transaction_date/README.rst

```diff
@@ -0,0 +1,4 @@
+Make sure that when reconciliation moves are generated for a bank
```

Comment:
Please use the OCA template from https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst